script.js

```diff
@@ -38,18 +38,18 @@ const iconToast = document.querySelector(".icon-toast-container");
 const iconToastClose = document.querySelector('.icon-toast-close');
```

Comment:
Using id selector will be better than class selector in Js

Reply:
Okay.  Will keep that in mind! Thanks